File: java/mediator/src/main/Student.java
package main;

public class Student extends CourseMember{

    public String Name;

    public Student(Mediator mediator)
    {
        super(mediator);
    }

    public void RecieveImage(String url)
    {
        System.out.println("Student " +  this.Name + " received image: " + url);
    }

    public void ReceiveAnswer(String answer)
    {
        System.out.println("student " + this.Name + " received answer " + answer);
    }

}

File: csharp/Mediator/Student.cs

namespace Mediator;

public class Student(Mediator mediator) : CourseMember(mediator)
{
    public string Name { get; set; }

    public void RecieveImage(string url)
    {
        Console.WriteLine("Student " +  this.Name + " received image: " + url);
    }

    public void ReceiveAnswer(string answer)
    {
        Console.WriteLine("student " + this.Name + " received answer " + answer);
    }
}
